### ISSUE TITLE
Multiple instance of Standalone button 

### ISSUE THREAD

**Issue by asimshazad** [UniSharp/laravel-filemanager#374]
I have installed latest version of the package with Laravel 5.4 and it is working fine.
All i need to know how can i use multiple instance of standalone buttons.
It only uploads a file to first filed regardless which field i have selected. 

Please have a look at this screenshot.
Secondly how can i find the path of the thumb,do i need to append "thumb" manually with store image path ?

![screen shot 2017-07-26 at 12 09 14 pm](https://user-images.githubusercontent.com/14991207/28608912-e10b3000-71fb-11e7-84cb-8b4023dd3d07.png)

**Comment by g0110280** [comment]
@asimshazad Yes, you should append "thumbs" to get thumbnail's path. We can only return one path, original image or thumbnail, at a time. Would you suggest how we can enhance this feature in the future?

**Comment by asimshazad** [comment]
Yup,i did it using creating a helper function,and appending a thumb in the image path.
This is what i did.

 ```
static function getThumb($path)
    {
        $str_to_insert = '/thumbs';
        $oldstr = $path;
        $pos = strrpos($path,"/");

         return substr_replace($oldstr, $str_to_insert, $pos, 0);
    }
```
Definitely you people can do this in more efficient way.

**Comment by BAKTETE** [comment]
is it implemented in version 1.9 and could you provide an example on how to approach this ?

**Comment by tanle25** [comment]
try
`$thumb_path =  Str::replaceLast(basename('$image_path'),'','$image_path')`
`$thumb_path .= 'thumbs'. basename('$image_path')`
